CONTRIBUTING.md

```diff
@@ -13,13 +13,130 @@ resources to make it easier to get your contribution accepted.
 
 This is a rough outline of what a contributor's workflow looks like:
 
-- Create a topic branch from where you want to base your work. This is usually master.
-- Make commits of logical units and add test case if the change fixes a bug or adds new functionality.
+### Step 1: Fork in the cloud
```

Comment:
I think we don't need to tell how to use Git here.

Reply:
It's ok, I close this pr.